tool_quality_checks/check_sewerage.py

```diff
@@ -5,13 +5,15 @@
 
 from OASDGLDatachecker.tool_quality_checks import sql_checks
 from OASDGLDatachecker.tool_quality_checks.sql_views import sql_views
-from OASDGLDatachecker.tool_quality_checks.sql_background_views import sql_background_views
+from OASDGLDatachecker.tool_quality_checks.sql_model_views import (
+    sql_understandable_model_views,
+)
 
 logger = logging.getLogger(__name__)
 
 
-def quality_checks(db, settings):
-    """Overall function for checking our model data"""
+def check_sewerage(db, settings):
+    """Overall function for checking our sewerage model data"""
```

Comment:
Wat bij zo'n check-functie handig is: aangeven wat 'ie doet als er iets niet klopt.
Knalt 'ie eruit met een exception?
Returnt hij een lijst met problemen?
Fixt hij de problemen?